cedars/app/db.py

```diff
@@ -896,15 +957,19 @@ def update_annotation_date(annotation_id, new_date):
     Args:
         annotation_id (str) : Unique ID for the annotation.
         new_date (str) : The new value to update the event date of an annotation with.
-            Must be in the format YYYY-MM-DD .
+            Must be in the format YYYY-MM-DD.
     Returns:
         None
     """
     # TODO: UTC dates
     logger.debug(f"Updating date on annotation #{annotation_id} to {new_date}.")
     date_format = '%Y-%m-%d'
     datetime_obj = datetime.strptime(new_date, date_format)
-    mongo.db["ANNOTATIONS"].update_one({"_id": ObjectId(annotation_id)},
+    anno = mongo.db["ANNOTATIONS"].find_one({"_id": ObjectId(annotation_id)})
```

Comment:
make sure the annotation is handled with the new change where we want to add annotation of the event date to the patient